lib/clAdapters/google-mail/index.js

```diff
@@ -143,14 +143,46 @@ var getAccessToken = function(clientId, adminEmail, userEmail, privateKey) {
   });
 };
 
+var getMoreEmails = function(messages, userEmail, token, apiVersion, additionalFields, emailRequestOptions, firstUri, nextPageToken) {
```

Comment:
Also, it might help to try to stick to a somewhat shorter line length, as many lines in this file are cut off in the github ui